liquibase-couchbase/src/test/java/liquibase/ext/couchbase/executor/service/TransactionExecutorServiceTest.java

```diff
@@ -0,0 +1,25 @@
+package liquibase.ext.couchbase.executor.service;
+
+import com.couchbase.client.java.Cluster;
+import org.junit.jupiter.api.Test;
+
+import static liquibase.ext.couchbase.configuration.CouchbaseLiquibaseConfiguration.IS_REACTIVE_TRANSACTIONS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class TransactionExecutorServiceTest {
+
+    private final Cluster cluster = mock(Cluster.class);
+
+    @Test
+    void Should_return_PlainTransactionExecutorService_or_ReactiveTransactionExecutorService() {
+        TransactionExecutorService transactionExecutorService = TransactionExecutorService.getExecutor(cluster);
+
+        if (IS_REACTIVE_TRANSACTIONS.getCurrentValue()) {
```

Comment:
I think, if this is a unit test we shouldn't rely on the specific value of external configuration property, it could also be mocked in the scope of the test